content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
           │      dvc exp pull     │
           ▼                       ▼
   ┌─────────────────┐    ┌────────────────┐
-  │   Code and      │    │                │
-  │   metafiles     │    │   Cached data  │  Local project
+  │    Code and     │    │      Data      │
+  │    metafiles    │    │    (cached)    │  Local project
   └─────────────────┘    └────────────────┘
 ```
 
-[dvc experiments]: /doc/user-guide/experiment-management/experiments-overview
-[any other commit]:
-  /doc/start/data-management/data-versioning#storing-and-sharing
-[git remotes]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+[remote storage]: /doc/user-guide/data-management/remote-storage
+[sharing-data]: /doc/start/data-management/data-versioning#storing-and-sharing
+
+If you need to share <abbr>cached</abbr> data that is tracked by DVC, [configure
+remote storage] first (e.g. Amazon S3 or SSH).
 
-### Preparation
+[configure remote storage]:
+  https://dvc.org/doc/user-guide/data-management/remote-storage#configuration
 
 <admon type="tip">
 
-If you don't need to share <abbr>cached</abbr> data, you can skip `dvc remote`
-configuration by using the `dvc exp push --no-cache` and
-`dvc exp pull --no-cache`.
+Check that you have the necessary remotes with `git remote -v` and (optionally)
+`dvc remote list`.
```

Comment:
p.s. I'd probably remove most of these tips (already documented in other guides) to make this section simpler, esp. if we do leave it on top (as method # 1).